docs/contributing/index.rst

```diff
@@ -89,15 +89,35 @@ versions are maintained through the community. Divio provides long term support
 `commercial support <http://divio.ch/en/commercial-support/>`_.
 
 
+.. _branch_policy:
+
 *************
 Branch policy
 *************
 
-* **master**: this is the current stable release, the version released on PyPI.
-* **support/3.2.x**: this will be our *next stable release*; this is the most
-  appropriate branch for fixes and patches that will go into the next **master**
-* **develop**: this will be *django CMS 3.3*; this is the most appropriate
-  branch for more substantial features that will need team co-ordination
+We maintain a number of branches on our GitHub repository.
+
+
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI.
+
+    **Fixes and backwards-compatible improvements** (i.e. the vast majority of pull requests) should be made against
```

Comment:
You could clarify how these changes make it to the "develop" branch (i.e. the committer cherry-picks or merges it)? This is different from Django's workflow where all PRs go to master and only those issues that qualify for a backport to the stable release are cherry-picked by the committer.